apps/opik-backend/src/main/java/com/comet/opik/utils/JsonUtils.java

```diff
@@ -50,6 +51,14 @@ public static JsonNode getJsonNodeFromString(@NonNull String value) {
         }
     }
 
+    public static JsonNode getJsonNodeFromStringWithFallback(@NonNull String value) {
+        try {
+            return getJsonNodeFromString(value);
+        } catch (Exception e) {
```

Comment:
@thiagohora - exception is silenced, we need to log is as an error at least to know whats happening...